Source/nthread.cpp

```diff
@@ -85,6 +85,9 @@ BOOL nthread_recv_turns(BOOL *pfSendAsync)
 		last_tick += 50;
 		return TRUE;
 	}
+#ifdef PLATFORM_CTR

```

Comment:
The `PLATFORM` defines are specific to StormLib. Please use `__3DS__` instead.
Also why is this particular one necessary?

Reply:
Using the Makefile.3ds, linking failed due to 'SNetReceiveTurns' not being defined.